covidapi/api/static/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Python API</title>
```

Comment:
We could add some CSS here to make this look nicer but I think it would be better to do that as another PR. Does that make sense to you @andreagrandi?

Reply:
@MatMoore yes, it makes sense to do that in a different PR. I wonder if there are any free templates we can borrow from, so we don't have to invest to much time into any design.